index.html

```diff
@@ -38,17 +41,180 @@ <h1 >
+                    A daily selection of privately personalized reads; no accounts
+                    or sign-ups required.
+                </p>
+                <ul class="technology-tags">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Javascript</li>
+                </ul>
+                <button class="See-project-button" type="button">See Project</button>
+            </div>
+
+
+
+
+
+        </div>
+    </div>
+    <!--about mew-->
+    <div class="about-me-wrap">
+        <h2>About Myself</h2>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my
+            work
+            and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <p class="lets-connect">
+            LET'S CONNECT
+        </p>
+        <ul class="social-icon-container about-me-icons">
+            <li><a class="github-button" href="#"><img class="social-icon" src="./icons/github-icon.png"
+                        alt="github button"></a></li>
+            <li>
+                <a class="linkedin-button" href="#"><img class="social-icon" src="./icons/Linkedin iconin-icon.png"
+                        alt="linked in button"></a>
+            </li>
+            <li><a class="twitter-button" href="#"><img class="social-icon" src="./icons/tw-icon.png"
+                        alt="twitter button"></a></li>
+            <li><a class="hand-button" href="#"><img class="social-icon" src="./icons/hand-icon.png"
+                        alt="a hand button"></a></li>
+            <li><a class="m-button" href="#"><img class="social-icon" src="./icons/m-icon.png" alt="m button"></a></li>
+        </ul>
+        <button class="getResumeButton" type="button">Get my Resume</button>
+        <div class="skill-list-container">
+            <figure>
+                <figcaption class="skill-section">Languages <a href="#"><img class="skill-icon"
+                            src="./icons/Disabledbutton-down-arrow.png" alt="show more button"></a> </figcaption>
+                <ul>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1js-icon.svg" alt=""><span>Javascript</span></li>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1html-icon.svg" alt=""><span>HTML</span></li>
+                    <li><img class="skill-icon" src="./icons/Ellipse 1css-icon.svg" alt=""><span>CSS</span></li>
+                </ul>
+            </figure>
+            <figure class="skill-section">
+                <figcaption>Frameworks</figcaption>
+                <ul>
+                    <ul>
+                    </ul>
+                </ul>
+            </figure>
+            <figure class="skill-section">
+                <figcaption>Skills</figcaption>
+                <ul>
+                </ul>
+            </figure>
+
+        </div>
+
+    </div>
```

Comment:
[OPTIONAL] Same for the About me section, Kindly I suggest you use the section tag for the main container of works-wrap, to follow HTML semantics and best practices. Also, I suggest you remove the white space in your code to make it clean and more professional. 😄